### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by johanwardaddy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,6 @@ fuck the queen
 
 penambahan untuk mencari konflik
 penambahan untuk mencari konflik
-penambahan untuk mencari konflik
-penambahan untuk mencari konflik
+
```
Comment (trihargianto) [inline README.md:16]: what the hell are you doing here???

Comment (johanwardaddy) [reply README.md:16]: saya menghilangkan beberapa kalimat

Comment (trihargianto) [reply README.md:16]: oke rapopo

**Review by trihargianto — APPROVED** [review]
LGTM!

**Pull request merged**